docs/sources/trackers/web-trackers/cross-domain-tracking/index.md

```diff
@@ -4,23 +4,239 @@ date: "2022-08-30"
+  }
+}
+```
+
+If you have a shared configuration deployed across multiple sites, you may want to check both:
+
+```javascript
+{
+  crossDomainLinker: function (linkElement) {
+    var networkSites = [
+      "mysite.com",
+      "sistersite.com"
+    ];
+    return linkElement.hostname !== location.hostname && networkSites.indexOf(linkElement.hostname) > -1;
+  }
+}
+```
+
+Here, link decoration will only occur when links are to different sites within the list.
+
+## Update event listeners
+
+When the tracker loads it doesn't immediately decorate links. Instead, it adds event listeners to links which decorate them when a user clicks on them or navigates to them using the keyboard. This ensures that the timestamp added to the querystring is fresh.
+
+If further links get added to the page after the tracker has loaded, you can use the tracker's `crossDomainLinker` method to add listeners again. Listeners won't be added to links which already have them so it's good practice to make sure you use the same linker function every call, as it won't be updated.
+
+<Tabs groupId="platform" queryString>
+  <TabItem value="js" label="JavaScript (tag)" default>
+
+```javascript
+snowplow('crossDomainLinker', function (linkElement) {
+  return (linkElement.href === 'http://acme.de' || linkElement.id === 'crossDomainLink');
+});
+```
+
+  </TabItem>
+  <TabItem value="browser" label="Browser (npm)">
+
+```javascript
+crossDomainLinker(function (linkElement) {
+  return (linkElement.href === 'http://acme.de' || linkElement.id === 'crossDomainLink');
+});
+```
+
+  </TabItem>
+</Tabs>
+
+
+## Configure extended format properties
+
+To decorate links with extended `_sp` format properties, set `useExtendedCrossDomainLinker`:
+
 <Tabs groupId="platform" queryString>
   <TabItem value="js" label="JavaScript (tag)" default>
 
-<CrossDomain lang="javascript" />
+```javascript
+snowplow('newTracker', 'namespace', '{{collector_url_here}}', {
+  appId: 'my-app-id',
+  appVersion: '0.1.0',
+  cookieSameSite: 'Lax',
+  crossDomainLinker: function(defineWhichLinksToDecorate) { },
+  useExtendedCrossDomainLinker: true,
+});
+```
 
   </TabItem>
   <TabItem value="browser" label="Browser (npm)">
 
-<CrossDomain lang="browser" />
+```javascript
+newTracker('namespace', '{{collector_url_here}}', {
+  appId: 'my-app-id',
+  appVersion: '0.1.0',
+  cookieSameSite: 'Lax',
+  crossDomainLinker: function(defineWhichLinksToDecorate) { },
+  useExtendedCrossDomainLinker: true,
+});
+```
 
   </TabItem>
 </Tabs>
+
+This will add `sessionId` and `sourceId` to the querystring.
+
+To include further properties:
+
+<Tabs groupId="platform" queryString>
+  <TabItem value="js" label="JavaScript (tag)" default>
+
+```javascript
+snowplow('newTracker', 'namespace', '{{collector_url_here}}', {
+  appId: 'my-app-id',
+  appVersion: '0.1.0',
+  cookieSameSite: 'Lax',
+  crossDomainLinker: function(defineWhichLinksToDecorate) { },
+  useExtendedCrossDomainLinker: {
+    userId: true;
```

Comment:
Maybe a comment here to state if your `userId` contains personal data to make sure your `crossDomainLinker` won't leak it to third party sites.